client/lib/domains/email-forwarding/index.js

```diff
@@ -0,0 +1,12 @@
+function emailForwardingPlanLimit( product_id ) {
+	let planLimit;
+	if ( product_id === 1008 ) {
```

Comment:
In general, we try to avoid including a product ID or slug at a random place in the code like this, in favor of functions that do the number/string comparison for us. This makes it easier for us to change the product ID or slug, as we only need to modify the function that does the actual comparison.

Currently, we use `isBusiness` for this, which lives in `lib/products-values`. If you change this function so that it accepts a product (which, in this case, is the plan), you could use that function instead of this comparison (i.e. `isBusiness( plan )`).